entries/d/deltadental.com.json

```diff
@@ -0,0 +1,51 @@
+{
+  "Delta Dental": {
+    "domain": "deltadental.com",
+    "img": "deltadental.com.png",
+    "tfa": [
+      "sms"
+    ],
+    "additional-domains": [
+      "deltadentalcoversme.com",
+      "deltadentalar.com",
+      "deltadentalks.com",
+      "deltadentalok.org",
+      "deltadentalpr.com",
+      "kentucky.deltadental.com",
+      "northcarolina.deltadental.com",
+      "tn.deltadental.com",
+      "www.deltadentalaz.com",
+      "www.deltadentalco.com",
+      "www.deltadentalct.com",
+      "www.deltadentalia.com",
+      "www.deltadentalid.com",
+      "www.deltadentalil.com",
+      "www.deltadentalin.com",
+      "www.deltadentalins.com",
+      "www.deltadentalma.com",
+      "www.deltadentalmi.com",
+      "www.deltadentalmn.org",
+      "www.deltadentalmo.com",
+      "www.deltadentalne.org",
+      "www.deltadentalnj.com",
```

Comment:
This website times out for me

Reply:
Hm, the New Jersey (`*nj`) one? These all successfully load for me